src/Sylius/Component/Core/spec/Resolver/EligibleDefaultShippingMethodResolverSpec.php

```diff
@@ -13,6 +13,7 @@
 
 namespace spec\Sylius\Component\Core\Resolver;
 
+use InvalidArgumentException;
```

Comment:
Not neede, I suppose

Reply:
Usually I add use statements for all imported classes. If this line is removed I would use `\ InvalidArgumentException` in the code, which the PhpStorm inspections complain about just as it would with something like `\Sylius\Component\Order\Model\Order` directly in the code instead of imported as `use Sylius\Component\Order\Model\Order`